app/views/fragments/promotion/pricingCta.scala.html

```diff
@@ -0,0 +1,15 @@
+@import com.gu.subscriptions.DigipackCatalog
+@import com.gu.memsub.promo.Promotion.AnyPromotion
+@import com.gu.memsub.promo.PromoCode
+@import model.DigitalEdition
+
+@(edition: DigitalEdition, catalog: DigipackCatalog, promoCode: PromoCode, promotion: AnyPromotion)
+<div class="pricing-cta__pricing">
```

Comment:
Since `__pricing` is BEM child syntax, I'd expect this to be within a `.pricing-cta` element